Tasks/quotes.js

```diff
@@ -1,30 +1,26 @@
+'use strict';
 // Change double quotation to open or close quotation
 
-const EMPTY = '';
+const quotes = (str) => {
+  const res = [];
```

Comment:
You can create an array with `new Array()` or `new Array(str.length)`

Reply:
Yeah, I think it would be better